api/GlobalEventHandlers.json

```diff
@@ -4281,7 +4281,7 @@
               "version_added": true
             },
             "edge": {
-              "version_added": "≤79"
+              "version_added": false
```

Comment:
Can you check why this was set to supported before?

Reply:
When Blink-versions of Edge had come out, all of the existing Edge data set to `true` and `null` was converted into ranges of `≤18` and `≤79` respectively, as per mirroring from Chrome Desktop.  Alas, the tests here have confirmed no support in Edge, so this is more like setting `null` to `false`.

Reply:
I see, thanks!